.github/workflows/release_java.yml

```diff
@@ -168,6 +168,10 @@ jobs:
             ~/osx-x86_64-local-staging/staging \
             ~/osx-aarch_64-local-staging/staging
 
+          tree -a $LOCAL_STAGING_DIR/staging
+
```

Comment:
nit: make it part of the Python script, perhaps a debug function and called at last.

Reply:
Addressed